### PR TITLE
Use spotless for formatting the bindings

### PR DESCRIPTION
**Pull request by Berstanio**
This should make generating the bindings a lot easier. Importing a formating file into IntelliJ is not that convinient and reliable.
On the initial commit there will be minimal formating changes (mainly comment line length), but after that spotless should make the formating way easier and consistent.

### CODE REVIEW

File: moe.apple/moe.platform.ios/Eclipse_Adjusted_NatJGen.xml [hunk 1]
```diff
@@ -0,0 +1,380 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Noisyfox) [inline moe.apple/moe.platform.ios/Eclipse_Adjusted_NatJGen.xml:1]: How was this file created?

**Comment by Noisyfox** [comment]
Overall LGTM, just need a bit more info on how the format file was written, and maybe add a link to the spotless doc in the readme?

**Comment by Berstanio** [comment]
> just need a bit more info on how the format file was written

The new file is just the old file in a updated file format. I retrieved it by importing the old file in IntelliJ and than export it directly again. The only manual adjustment was disabling the formatting of the header: org.eclipse.jdt.core.formatter.comment.format_header

> add a link to the spotless doc in the readme?

In which readme? UPDATE-IOS-BINDING.md?

**Comment by Noisyfox** [comment]
Yeah I think just add a line stating this file is exported from IntelliJ should be sufficient.

**Review by Noisyfox — APPROVED** [review]

**Review by Noisyfox — APPROVED** [review]

**Pull request merged**